src/Symfony/Bridge/PhpUnit/composer.json

```diff
@@ -18,7 +18,8 @@
     "require": {
         "php": ">=5.5.9 EVEN ON LATEST SYMFONY VERSIONS TO ALLOW USING",
         "php": "THIS BRIDGE WHEN TESTING LOWEST SYMFONY VERSIONS.",
-        "php": ">=5.5.9"
+        "php": ">=5.5.9",
+        "symfony/deprecation-contracts": "^2.1"
```

Comment:
We cannot do this because of the PHP version. Let's call @trigger_error() directly instead.